app/controllers/schemes_controller.rb

```diff
@@ -170,14 +174,28 @@ def scheme_params
                                                      :intended_stay,
                                                      :confirmed)
 
-    full_params = required_params[:arrangement_type] == "D" && required_params[:owning_organisation_id].present? ? required_params.merge(managing_organisation_id: required_params[:owning_organisation_id]) : required_params
+    if arrangement_type_changed_to_different_org?(required_params)
+      required_params[:managing_organisation_id] = nil
```

Comment:
Is there any scenario where this could be set but to a different id than the owning org that needs checking for?

Reply:
Hm, not sure I understand, can you rephrase? or give an example?

Reply:
We're checking if the `arrangement_type` is that the organisation providing support services is separate from the organisation owning the stock?

And our check is just for nil, should it also check for the scenario where a managing_org_id param is present but doesn't match the owning_org_id or should that never happen?